### ISSUE TITLE
Feature request: copy/pasting multiple notes in Edit Mode

### ISSUE THREAD

**Issue by SmilleyMLK** [UltraStar-Deluxe/USDX#412]
More specifically their length, pitch, relative distance and features like lyrics, fade-out or golden notes. It would help a lot with refrain/chorus parts and repeated sequences.
Alternatively, being able to overlay earlier lines as transparent tiles, so you can match up the new notes.

**Comment by dgruss** [comment]
this is already supported with ctrl + 2-4 etc. and documented in the help

**Comment by barbeque-squared** [comment]
Yeah I don't know the exact shortcuts but I have definitely pasted entire/half sentences in the past as well. Closing this for now in an attempt to clean up the issuelist a bit, if it's still an issue either reopen or just make a new one. Maybe also be a little bit more specific (screenshots always help) like "it would be great if I could select this particular note and if I then go to somewhere else in the song, what do you expect to happen.

This works especially well if you also have an idea about at least _some_ of the edge cases, like what happens if the second note is in the middle of a sentence etc.
